### PR TITLE
Fixed 'learn' page bugs

### PR DESCRIPTION
**Pull request by rudranshg**
Fixed responsive bugs on the learn page (tree-node)
Added gradient background
Imported new fonts to the site

### CODE REVIEW

**Review by sps1112 — APPROVED** [review]
Seems alright.

File: css/style.css [hunk 1]
```diff
@@ -4,6 +4,24 @@
  *
  */
  @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+ @import url('https://fonts.googleapis.com/css2?family=Oxygen&display=swap');
+ @import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@500&display=swap');
+
+body {
+  background-color: #f8f9d2;
+  background-image: linear-gradient(315deg, #f8f9d2 0%, #e8dbfc 74%);
```
Comment (sps1112) [inline css/style.css:12]: I don't think that we need a gradient background. 🤔

**Review by sps1112 — CHANGES_REQUESTED** [review]
You can remove the gradient background for now

File: css/style.css (hunk 1 above)
Comment (rudranshg) [reply css/style.css]: Sure.

**Comment by rudranshg** [comment]
Done. Removed the gradient for now.

**Review by sps1112 — APPROVED** [review]
👍🏼

**Pull request merged**